blog/wyniki-badania-plac-w-komunikacji-technicznej-2023/index.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: 'Wyniki badania płac w komunikacji technicznej 2023'
+authors: [pensjonatus, mojk]
+date: '2024-05-15'
+tags:
+  - 'badanie-plac'
+  - 'praca'
+  - 'ankieta'
+  - 'kariera'
+  - 'praca'
+  - 'spolecznosc'
+coverImage: 'Badanie_plac_2023_wyniki.png'
+---
+
+import YearlyComparisonChart from '@site/src/components/Survey/YearlyComparison/Chart';
+import SurveyTable from '@site/src/components/Survey/SurveyTable';
+import SurveyBarChart from '@site/src/components/Survey/SurveyBarChart';
+import SurveyBulletedList from '@site/src/components/Survey/SurveyBulletedList';
+import SurveyPieChart from '@site/src/components/Survey/SurveyPieChart';
+import { companySizeTableProps } from './data';
+
+Najwyższa pora, by opublikować wyniki corocznej ankiety dotyczącej płac w
+komunikacji technicznej. Szczegóły poniżej.
+
+{/* truncate */}
+
+W tym roku postanowiliśmy wejść na wyższy poziom interakcji. W związku z tym, w artykule znajdziecie bardziej interaktywne wykresy i tabelki.
+Mamy nadzieję, że przypadną Wam do gustu bardziej niż statyczne zrzuty ekranu, których używaliśmy w poprzednich latach.
+
+## O co pytaliśmy?
+
+W tym roku większość pytań była taka sama jak w latach poprzednich:
+
+1. Na jakim stanowisku pracujesz?
+2. W którym województwie jest siedziba Twojej firmy?
+3. Ile wynosi Twój staż pracy w branży komunikacji technicznej?
+4. Jakie masz przygotowanie do pracy w branży?
+5. Pracujesz w filmie polskiej czy międzynarodowej?
+6. Jaka jest Twoja forma zatrudnienia?
+7. Jak duża jest Twoja firma?
+8. Jakiego rodzaju dokumentację tworzysz?
+9. Ile wynosi Twoje miesięczne wynagrodzenie brutto?
+10. Jak bardzo jesteś zadowolony/a ze swojej pracy w skali od 1 do 5 ?
+11. Jakie jest Twoje główne wykształcenie?
+
+Dodatkowo, pojawiły się dwa nowe pytania:
+
+1. Czy dla Twojego wynagrodzenia zastosowanie mają autorskie koszty uzyskania
+  przychodu (tzw. tax break)?
+2. W jakim modelu pracujesz - zdalnie czy hybrydowo?
+
+## Porównanie z poprzednimi latami
+
+Jeśli ciekawi Was co zmieniło się od naszego pierwszego badania, sprawdźcie zestawienie poniżej.
+
+<YearlyComparisonChart />
+
+## Pierwsze wnioski
+
+- W tym zebraliśmy aż 133 odpowiedzi! Bardzo nas cieszy, że tak wiele osób jest
+  zainteresowanych naszą ankietą. Ciekawe jaki to procent wszystkich tech writerów w Polsce.
+  🤔
+- Zarobki nadal mają tendencję wzrostową. 💰
+
+## Profesje
+
+Podobnie jak w latach poprzednich, większość osób biorących udział w ankiecie
+identyfikuje się jako Technical Writer (specjalista ds. dokumentacji
+technicznej). Pojawił się też bardzo ciekawy tytuł "Content Wrangler", ale
+zakwalifikowaliśmy go jako "technical writer".
+
+W odpowiedziach pojawiały się też feminatywy z jednoczesnym
+spolszczeniem, takie jak "technical writerka". Jako znawcy języka, jesteśmy bardzo ukontentowani tym faktem.
```

Comment:
w tych dwoch akapitach proponuje wybrac pisanie zagranicznych i polskich okreslen profesji w jeden sposob. Najlepiej wszystkie z duzej, bo to wazne slowa w tym artykule.

Reply:
mozecie tez wziac pod uwage jaka konwencje uzyjecie w tresci samych tabel.